src/remotepublish.cmd.js

```diff
@@ -155,6 +158,30 @@ class RemotePublishCommand extends AbstractCommand {
     return this;
   }
 
+  withCustomVCL(value) {
+    if (value && value.length > 0) {
+      const vcl = {};
+      value.forEach((file) => {
+        try {
+          const fullPath = path.resolve(file);
+          const name = path.basename(fullPath, '.vcl');
+          const content = fs.readFileSync(fullPath).toString();
+          vcl[name] = content;
+        } catch (error) {
+          this.log.error(`Cannot read provided custom vcl file ${file}`);
+          throw error;
+        }
+      });
+      return this.withVCL(vcl);
+    }
+    return this;
+  }
+
+  withVCL(value) {
```

Comment:
- no jsdoc
- what's the purpose of this in contrast to `withCustomVCL` ? do you really need both?
- needs to be an object, so name it `withVCLs`